cpp/configs/contribute_example.cfg

```diff
@@ -16,6 +16,11 @@ maxSimultaneousGames = Fill in a number here
 watchOngoingGameInFile = false
 watchOngoingGameInFileName = watchgame.txt
 
+# Set to true if you want one of KataGo's games to be output as JSON, so you can programatically process/visualize them
```

Comment:
Do you mind if I delete all of this in the example cfg? Seems not so useful for a common user to be configuring the option to dump an enormous amount of data or not to stdout. KaTrain can pass it using --override-config where needed when launching the program.